src/scss/components/choice-tile/_index.scss

```diff
@@ -0,0 +1,218 @@
+@import "components/box/config";
+@import "components/grid/config";
+@import "components/form/mixins/input";
+@import "./mixins/hxChoiceControl";
+@import "mixins";
+
+hx-tile {
+  @include box-size(md);
+
+  border-radius: 2px;
+  display: inline-block;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  > header {
+    font-size: 1rem;
+    font-weight: 700;
+    margin-bottom: 0;
+  }
+
+  > p {
+    font-size: 0.75rem;
+  }
+
+  &.hxSm {
+    $pad: 0.75rem;
+
+    padding: $pad;
+
+    > p {
+      display: none;
+    }
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  &.hxLg {
+    $pad: 1.75rem;
+
+    padding: $pad;
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  > header + *,
+  .hx-tile-icon + * {
+    margin-top: 0.5rem;
+  }
+
+  .hx-tile-icon {
+    hx-icon {
+      font-size: 1.75rem;
+    }
+  }
+
+  hx-icon[type="checkmark"] {
+    display: none;
+    font-size: 1rem;
+    right: 1.25rem;
+    top: 1.25rem;
+  }
+
+  &.hxSubdued {
+    @include hxChoiceTile--subdued;
+
+    &:hover {
+      @include hxChoiceTile--subdued($pseudo-state: hover);
+    }
+  }
+}
+
+.hxRow > label.hxChoice {
+  @include grid-column;
+}
+
+// Layout & Geometry
+// ========== COMMON ==========
+label.hxChoice {
+  position: relative;
+}
+
+label.hxChoice > input {
+  @include hxInput(visuallyHidden);
+
+  &:focus {
+    @include reset-focus;
+  }
+
+  &:checked + hx-tile {
+    hx-icon[type="checkmark"] {
+      display: block;
+      position: absolute;
+    }
+  }
+}
+
+// ========== SUBDUED ==========
+label.hxChoice > input + hx-tile:hover.hxSubdued {
+  @include hxChoiceTile--subdued($pseudo-state: hover);
+}
+
+label.hxChoice > input[disabled] + hx-tile.hxSubdued,
+label.hxChoice > input[invalid] + hx-tile.hxSubdued {
+  opacity: 1;
+}
+
+// ========== DEFAULT ==========
+// NOTE: These styles may be able to go into default hx-tile defs
+label.hxChoice > input + hx-tile {
+  background-color: $gray-0;
+  border: solid 1px $gray-400;
+  color: $gray-900;
+  cursor: pointer;
+
+  .hx-tile-icon {
+    color: $gray-700;
+  }
+}
+
+label.hxChoice > input + hx-tile:hover {
+  @include hxChoiceTile($pseudo-state: hover);
+}
+
+// same as hover, with focus glow
+label.hxChoice > input:focus + hx-tile {
+  @include hxChoiceTile($pseudo-state: focus);
+}
+
+// ========== CHECKED ==========
+label.hxChoice > input:checked + hx-tile {
+  border-color: $cyan-900;
+  color: $cyan-900;
+
+  .hx-tile-icon {
+    color: $cyan-900;
+  }
+
+  > p {
+    color: $gray-900;
+  }
+
+  // DEPRECATED: remove in v1.0.0
+  hx-tile-description {
+    color: $gray-900;
+  }
+}
+
```

Comment:
Missing styles originally present in LESS... 

https://github.com/HelixDesignSystem/helix-ui/blob/master/src/less/components/choice-tile/index.less#L159-L165

Reply:
![image](https://user-images.githubusercontent.com/55139500/70634030-8528d680-1c57-11ea-9e02-8b487746e71d.png)

Actually this styles related to that so been removed. Dont see any stoppers. Please let me know if required to be added.

Reply:
Resolving comment, because hover and focus styles are the same regardless of checked state.